src/index.tsx

```diff
@@ -2,18 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import 'index.css';
 import App from 'components/App/App';
-import {store} from 'store/store';
-import {Provider} from 'react-redux';
+import { store } from 'store/store';
+import { Provider } from 'react-redux';
 import * as serviceWorker from './serviceWorker';
-import CssBaseline from "@material-ui/core/CssBaseline";
+import CssBaseline from '@material-ui/core/CssBaseline';
 import axios from 'axios';
 
 axios.interceptors.response.use(
-  response => {
+  (response) => {
     return response;
   },
-  function(error) {
-    if (error.response.status === 400) {
+  function (error) {
+    if (error.response.status >= 400 && error.reponse.status <= 422) {
```

Comment:
After receiving 403, I see this instead of an alert:
```
Uncaught (in promise) TypeError: Cannot read property 'status' of undefined
    at index.tsx:16
```
Does it have something to do with this code? I don't know, sorry.

Reply:
Another typo here) Fixed, thanks